Gemfile

```
@@ -4,6 +4,8 @@ gem 'rails', '~>3.2.22'
 
 # pin this to post-CVE 2017-5946
 gem 'rubyzip', '~> 1.2.2'
+# pin the following gems to work with Ruby 2.1 and Rails 3
+gem 'public_suffix','~>3.0.3'
```

```suggestion
gem 'public_suffix','3.0.3'
```
https://bundler.io/v1.5/gemfile.html#gemfiles
>The specifier ~> has a special meaning, best shown by example. ~> 2.0.3 is identical to >= 2.0.3 and < 2.1. ~> 2.1 is identical to >= 2.1 and < 3.0. ~> 2.2.beta will match prerelease versions like 2.2.beta.12.